src/reachy2_sdk/orbita/orbita.py

```diff
@@ -91,15 +92,15 @@ def __repr__(self) -> str:
     #     else:
     #         raise ValueError("pid should be of type Tuple[float, float, float]")
 
-    def get_speed_limit(self) -> Dict[str, float]:
+    def get_speed_limits(self) -> Dict[str, float]:
         """Get speed_limit of all motors of the actuator"""
```

Comment:
we should add in the comment the unit (percentage of the max speed right?)

Reply:
done